modules/workflow-service-impl/src/main/java/org/opencastproject/workflow/endpoint/WorkflowRestService.java

```diff
@@ -444,8 +444,8 @@ public Response getWorkflowsAsXml(@QueryParam("state") List<String> states, @Que
           @RestParameter(name = "seriesTitle", isRequired = false, description = "Filter results by series title", type = STRING),
           @RestParameter(name = "creator", isRequired = false, description = "Filter results by the mediapackage's creator", type = STRING),
           @RestParameter(name = "contributor", isRequired = false, description = "Filter results by the mediapackage's contributor", type = STRING),
-          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by workflow start date.", type = STRING),
-          @RestParameter(name = "todate", isRequired = false, description = "Filter results by workflow start date.", type = STRING),
+          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by event date.", type = STRING),
```

Comment:
It's actually the created date (so not e.g. the bibliographic start date, which I would expect), maybe add that here?

Reply:
```suggestion
          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by `created` Dublin Core metadata .", type = STRING),
```
?

Reply:
```suggestion
          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by event created date.", type = STRING),
```
?

Reply:
Actually, I take that back, I got confused by the use of "created" in the key, but I took a closer look at it just now and it's actually the bibliographic start date, not the created date. We have too many dates...

Reply:
„The bibliographic start date“ is the Dublin Core „created“ field, isn't it?
So… which suggestion do you want me to apply here?

Reply:
No, I think it's "temporal", actually.
(So "Start Date" in the event metadata is "temporal" in the catalog, and "Created" is "created"... which would actually make sense?)

Reply:
> No, I think it's "temporal", actually.

No, it's not. The DC field `temporal` contains start and end date information used by capture agents for recordings. Thats the technical start (and end) date. That's not what is being used by this endpoint. You can test this easily on http://develop.opencast.org since the uploaded events have no `temporal` metadata, only `created` and you can still use this field for filtering the workflows.


Are you happy with:
```suggestion
          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by the date contained in the `created` Dublin Core metadata. Use UTC dates in the format <pre>2020-12-24T08:30:00Z</pre>.", type = STRING),
```

Reply:
Okay, so I spent probably way too much time on this?

1. "temporal" contains the bibliographic start date, not the technical one. Reasoning:
 
The config mapping dublin core entries to event metadata (`org.opencastproject.ui.metadata.CatalogUIAdapterFactory-episode-common.cfg`) says so:

```
property.startDate.inputID=temporal
property.startDate.label=EVENTS.EVENTS.DETAILS.METADATA.START_DATE

property.created.inputID=created
property.created.label=EVENTS.EVENTS.DETAILS.METADATA.CREATED
```

And when I schedule an event via the UI and later change the bibliographic (!) start date, the temporal field contains that value, not the technical start date. Actually, the technical start date then isn't in the dublin core catalog at all - but you only see that when the two dates are not the same.

The events on develop are uploaded via a script, so that's probably why they don't have a temporal field - when I upload one via the UI, it does.

2. The workflow code is using the bibliographic start date, not the created date. The code says so:

WorkflowServiceSolrIndex:

    if (mp.getDate() != null) {
      doc.addField(CREATED_KEY, mp.getDate());
    }

(I know it says "created", but the Mediapackage Start Date is actually the bibliographic one:)

CommonEventCatalogUIAdapter:

     // Mediapackage start date is set by event metadata start date. The "created" metadata field is not used.
    MetadataField startDate = abstractMetadata.getOutputFields().get("startDate");
    mediaPackage.setDate(sdf.parse((String) startDate.getValue()));

Though it doesn't make much of a difference, since these dates are very often identical, and created is probably used as a fallback somewhere.

----------------------------------------------------
So I guess I'm still arguing for "temporal" here. :D

Reply:
Then, based on what you found out, let's go with:

```suggestion
          @RestParameter(name = "fromdate", isRequired = false, description = "Filter results by the creation date metadata field of the media package which is usually populated via the `temporal` Dublin Core element. Use UTC dates in the format <pre>2020-12-24T08:30:00Z</pre>.", type = STRING),
```